### PR TITLE
Improve CLI flags output

### PR DESCRIPTION
**Pull request by paulirish**
image

* provides the options for the `--output` flag
* ~~introduces `--json` shorthand flag~~

### CODE REVIEW

File: cli/index.js [hunk 1]
```diff
@@ -31,25 +31,36 @@ if (semver.lt(process.version, '5.0.0')) {
...
+    --output           Reporter for the results
+                       Reporter options: ${formatOptions}  (default=pretty)
+    --output-path      The file path to output the results (default=stdout)
+                       Example: --output-path=./lighthouse-results.html
+    --json             Shorthand for --output=json
 `);
 
 const url = cli.input[0] || 'https://pwa.rocks/';
-const outputMode = cli.flags.output || 'pretty';
+const outputMode = cli.flags.json ? 'json' : (cli.flags.output || 'pretty');
```
Comment (brendankenny) [inline cli/index.js]: should `json` win here? Or `cli.flags.output || (cli.flags.json ? 'json' : 'pretty')`

Comment (brendankenny) [inline cli/index.js]: feels weird to expose `OUTPUT_MODE` but not use `Printer.OUTPUT_MODE.pretty` and `Printer.OUTPUT_MODE.json`, but don't feel strongly either way

**Comment by brendankenny** [comment]
LGTM with two style things to consider

File: cli/index.js (hunk 1 above)
Comment (paullewis) [inline cli/index.js]: Yeah this weirds me out. If we feel like `json` should be default, we should switch to that. But to have --json = --output=json seems wonky, plus I had to read this ternary a few times.

Comment (paullewis) [inline cli/index.js]: Actually if we do this, we should use it with the formatters

**Comment by paulirish** [comment]
I'll remove --json for now. Not a huge deal now that the reporter/printer options are clear to the user.

**Comment by paulirish** [comment]
k all set.  image above updated.

**Pull request merged**